Project.toml

```diff
@@ -9,12 +9,18 @@ Bijectors = "76274a88-744f-5084-9051-94815aaf08c4"
 ChainRulesCore = "d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4"
 Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
 FastGaussQuadrature = "442a2c76-b920-505d-bb47-c5924d526838"
+JLD2 = "033835bb-8acc-5ee8-8aae-3f567f8a3819"
```

Comment:
this doesn't need to be in the Project.toml

Reply:
Sorry, do you mean to just remove it here, or the dependency entirely? If the latter, then I think this is related to the conversation here https://github.com/JuliaAstro/Transits.jl/pull/23#issuecomment-897007955

Reply:
right, but this is in the Project dependencies, not the test dependencies. You don't use JLD2 anywhere in the project source, so it's just an orphaned dependency here

Reply:
Ahh, totally didn't notice that this was for the main Project.toml and not the test's. Fixed: 06cb687